src/scripts/mutual_checker.json

```
@@ -6,5 +6,12 @@
     "color": "white",
     "background_color": "#777777"
   },
-  "help": "https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#mutual-checker"
+  "help": "https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#mutual-checker",
+  "preferences": {
+    "showOnlyMutuals": {
+      "type": "checkbox",
+      "label": "Only show mutuals on dashboard",
```

This is a nitpick, so feel free to resolve without committing this if you disagree, but I feel like the description here could stand to be a little more verbose...

```suggestion
      "label": "Only show posts from mutuals on dashboard",
```